assets/javascripts/src/actions.es6

```diff
@@ -28,6 +32,47 @@ export const GA_ENABLED = "GA_ENABLED";
 
 export const AUTOFILL = "AUTOFILL";
 
+export function openStripeCheckout(dispatch) {
+    const state = store.getState();
+    console.log('is this thing on');
+    let hide = () => {
```

Comment:
not sure having this here helps readability, probably just declare the lambda in the `showCheckout` call